packages/blade/src/components/BladeProvider/BladeProvider.tsx

```diff
@@ -1,15 +1,12 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
why this?

Reply:
https://github.com/razorpay/blade/pull/536#discussion_r926655298

Reply:
turn it off at root level?

Reply:
Will do.